app/controllers/admin/action_pages_controller.rb

```diff
@@ -51,7 +51,7 @@ def create
   end
 
   def edit
-    @actionPage.partner ||= Partner.new
+    @actionPage.partners ||= Partners.all
```

Comment:
I'm not sure this does anything since an uninitialized `has_many` relation is still non-nil (it's an empty ActiveRecord::Associations::CollectionProxy).